guides/release/configuring-ember/debugging.md

```diff
@@ -68,6 +68,9 @@ export default Application.extend({
 ```
 ### Dealing with deprecations
 
+In addition to what is described in the [Handling Deprecations guide](../handling-deprecations),
```

Comment:
Try this:
```suggestion
In addition to what is described in the [Handling Deprecations guide](./handling-deprecations),
```

I think it's `./` for siblings, `../` for going up a level on non-index pages, `../../` for going up a level on index pages. If that's right, we need to update CONTRIBUTING.md